repos/spack_repo/builtin/packages/py_astropy/package.py

```diff
@@ -21,6 +21,13 @@ class PyAstropy(PythonPackage):
     license("BSD-3-Clause", checked_by="lgarrison")
 
     version("7.0.1", sha256="392feeb443b2437cd4c2e0641a65e0f15ba791e148e9b1e5ed7de7dfcb38e460")
+    version("6.1.7", sha256="a405ac186306b6cb152e6df2f7444ab8bd764e4127d7519da1b3ae4dd65357ef")
```

Comment:
have you checked https://github.com/astropy/astropy/blob/main/pyproject.toml to see all the dependency constraints are met in in this package.py?

Reply:
in any case, have you had a chance to checkout pyproject.toml to see if any dependency constraints should be adjusted?

Reply:
@bnikolic can you please confirm all dependencies are satisfied via comparing pyproject.yaml against spack package.